code/modules/paperwork/paperbin.dm

```diff
@@ -74,6 +75,10 @@
 		bin_pen = null
 		update_icon()
 	else if(total_paper >= 1)
+		response = alert(user, "Do you take regular paper, or Carbon copy paper?", "Paper type request", "Regular", "Carbon-Copy", "Cancel")
```

Comment:
Keep spelling the same here: "carbon copy paper". Can the hyphenated version in the options be made not hypenated?